### DISCUSSION TITLE
Steaming audio quality

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#443] [Q&A]
I’ve seen some alternatives to audiogridder and they show the steaming quality you can select for the audio. I can’t find any info about how audiogridder processes the audio. Is it lossless or does the audio quality degrade in and out depending on the connection quality. 

I would like to find more information about this. 

I’m wondering like if I’m rendering a song export and if the connection dips doesn’t that mean that the quality of the export will also be affected. I would like to try this program but I only have one computer at the moment.

**Comment by CaBuffalo** [comment]
Yes, the audio export will be affected the your network connection dropped
during the exporting.

On Mon, Mar 22, 2021 at 2:16 PM bsjdsp ***@***.***> wrote:

> I’ve seen some alternatives to audiogridder and they show the steaming
> quality you can select for the audio. I can’t find any info about how
> audiogridder processes the audio. Is it lossless or does the audio quality
> degrade in and out depending on the connection quality.
>
> I would like to find more information about this.
>
> I’m wondering like if I’m rendering a song export and if the connection
> dips doesn’t that mean that the quality of the export will also be
> affected. I would like to try this program but I only have one computer at
> the moment.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/443>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BL72ERIT4WL4N2EOLTE6XTXANCNFSM4ZT43BJA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
AG does not modify the audio. It's lossless if you will. Only plugins on the server side will modify audio.

If you have network issues or your server is overloaded etc you might get audio drop outs or your DAW playback engine to stop.

> I’m wondering like if I’m rendering a song export and if the connection dips doesn’t that mean that the quality of the export will also be affected. I would like to try this program but I only have one computer at the moment.

If you render in realtime and you get drop outs, your printed track will be broken. If you render offline, the DAW can just wait for the data, so there is no drop outs. But of course if your network breaks entirely, there nothing AG can do.

**Reply by bsjdsp** [reply]
Is there a signal strength meter I can view at all times to ensure my tracks are getting good signal when listening and or exporting

**Reply by apohl79** [reply]
There is no signal strength. You either have a connection or not. If you have a connection, you don't loose data. If the latency is too high, your DAW might either skip audio data during playback (drop outs) or it stops, like what logic would do.

You can use the plugin monitor (check the plugin docu on the website) to monitor the latency and connection state of all your channels/instances.
